apps/reference/_supabase_js/generated/auth-admin-inviteuserbyemail.mdx

```diff
@@ -111,10 +111,11 @@ A URL or mobile address to send the user to after they are confirmed.
 
 - Requires a `service_role` key.
 - This function should only be called on a server. Never expose your `service_role` key in the browser.
```

Comment:
Instead of writing this everywhere, lets probably create an overview page and say these should only be server side and all of them require your service role key?